Alloy/commands/compile/ast/handle-alloy-globals.js

```diff
@@ -0,0 +1,98 @@
+const toCheck = [ 'Alloy', '_', 'Backbone' ];
```

Comment:
This variable name is ok, but may make more sense with a name like `ALLOY_GLOBALS` or the wordier `GLOBALS_TO_CHECK`? super nit-picky style thing.

Reply:
Agreed, went for a combination of the two with `ALLOY_GLOBALS_TO_CHECK` :)